### ISSUE TITLE
Accessibility + common gotchas

### ISSUE THREAD

**Issue by tjheffner** [phase2/frontend-docs#4]
our default patterns are (going to be) accessible out of the box. we should have a section on accessibility explaining what we mean by that/why. i don't think we necessarily need to break down each aria attribute, but a few good paragraphs, and some links back to relevant p2 blogs/podcasts and external helpful further reading should cover it. 

https://www.w3.org/TR/wai-aria-1.1/
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA
https://www.phase2technology.com/blog/accessibility-made-easy-introducing-holmes-module
https://www.phase2technology.com/blog/download-the-web-accessibility-playbook
etc. etc.
